Makefile

```diff
@@ -199,6 +200,22 @@ kill_port_forwardings:
 kill_all_port_forwardings:
 	scripts/utils.sh kill_port_forwardings '$(SERVICE_NAME) $(UI_SERVICE_NAME)'
 
+deploy_service_on_cluster:
```

Comment:
lets rename it to "deploy_service_on_ocp_cluster

Reply:
done